.github/workflows/tests.yml

```
@@ -192,3 +121,29 @@ jobs:
         shell: bash
         run: |
           exit $([[ "${{steps.get_version.outputs.IS_NEW_VERSION}}" = "true" ]] && echo 0 || echo 1)
+
+  docker-build:
+    name: 'Build docker image'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Docker build
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
+      - name: Setup Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: '--debug'
+      - name: Publish multi-arch image
```

```suggestion
      - name: Build multi-arch image
```
We aren't publishing.